lang/python/pytz/Makefile

```diff
@@ -15,30 +15,48 @@ PKG_LICENSE:=MIT
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/p/pytz
 PKG_HASH:=d5f05e487007e29e03409f9398d074e158d920d36eb82eaf66fb1136b0c5374c
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(BUILD_VARIANT)-pytz-$(PKG_VERSION)
 
-PKG_BUILD_DEPENDS:=python/host
 HOST_BUILD_DEPENDS:=python/host
 
 include $(INCLUDE_DIR)/host-build.mk
 include $(INCLUDE_DIR)/package.mk
 include ../python-package.mk
+include ../python3-package.mk
 
-define Package/pytz
+PKG_UNPACK:=$(HOST_TAR) -C $(PKG_BUILD_DIR) --strip-components=1 -xzf $(DL_DIR)/$(PKG_SOURCE)
+HOST_UNPACK:=$(HOST_TAR) -C $(HOST_BUILD_DIR) --strip-components=1 -xzf $(DL_DIR)/$(PKG_SOURCE)
+
+define Package/pytz/Default
   SUBMENU:=Python
   SECTION:=lang
   CATEGORY:=Languages
   MAINTAINER:=Gergely Kiss <mail.gery@gmail.com>
-  TITLE:=World timezone definitions, modern and historical
   URL:=https://sourceforge.net/projects/pytz/
-  DEPENDS:=+python
+endef
+
+define Package/pytz
```

Comment:
I know this isn't great to do, and may break some backwards compatibility, but let's rename this to `Package/python-pytz`
it's not great either way; one way breaks backwards compat, the other has people complaining with "why isn't this consistent ? it's stupid"

at least backwards compat complaints fade over time

otherwise, this is good

Reply:
Ok. It makes sense to me. I was just afraid to do so that it might be problem.